exercises/palindrome-products/test/test_palindrome_products.c

```diff
@@ -1,6 +1,8 @@
 #include "vendor/unity.h"
 #include "../src/palindrome_products.h"
 
+product_t *product;
```

Comment:
This is a globally visible symbol. Can you either make this `static` so that it's only visible within this file or simply push its declaration into each test? I don't know that there's a compelling reason to share this declaration.